### ISSUE TITLE
 How do I upsample a wav once the model is trained?

### ISSUE THREAD

**Issue by go-dustin** [mindslab-ai/nuwave#14]
I think this is a very interesting project and I'd like to test it but I'm not a data scientist. I see how to train and test but I don't see any examples of how to use it. I looked through the code but didn't see anything that gave me a clear indication on how to use it. How do I upsample a wav once the model is trained?

**Comment by junjun3518** [comment]
Hi! Thank you for your interest! 

We already wrote code about wav file upsampling (inference) with trained model, but we did not write it on README.
https://github.com/mindslab-ai/nuwave/blob/f941928c12db6d0e325e8c1997079d274ea4cede/sampling.py

Maybe we could update README during this week. 

Thank you for pointing out this.

**Comment by go-dustin** [comment]
I did see that script but it wasn't clear what it did. Please write up a readme and I'd be happy to test..!

Would it be possible to share the pretrained model so I can test it quickly without having to train.?

**Comment by freds0** [comment]
Yes, checkpoints available at https://github.com/mindslab-ai/nuwave/issues/18.
